examples/training/ms_marco/train_bi-encoder_mnrl.py

```diff
@@ -78,7 +78,9 @@
     word_embedding_model = models.Transformer(model_name, max_seq_length=max_seq_length)
     pooling_model = models.Pooling(word_embedding_model.get_word_embedding_dimension(), args.pooling)
     model = SentenceTransformer(modules=[word_embedding_model, pooling_model])
-
+    
+# Resize model embeddings to match curr tokenizer in case of added tokens
+model._first_module().auto_model.resize_token_embeddings(len(model.tokenizer))
```

Comment:
I would remove that line to keep the example simple. The model should match the tokenizer by how it it loaded/constructed.

Reply:
Okay you are right I removed it.